### DISCUSSION TITLE
Conveyor connector bandwith.

### DISCUSSION THREAD

**Discussion by RedShiftT** [voxeltycoon/issues#569] [General]
The bandwidth of conveyor connector depends of game speed or system performance.
![ezgif com-crop](https://user-images.githubusercontent.com/32413334/60718017-39e1fd80-9f2c-11e9-9dfe-51d870902b9a.gif)

Pentium 2020m, 2 cores, 2,4GHz
Intel HD Graphics

Game v. 0.71

**Comment by drewtato** [comment]
This also seems to happen with other places: between vehicles and storage, and on conveyor intersections. Loading and unloading cargo are especially inconvenient.

**Comment by Toys0125** [comment]
if I had to guess this has to do with physics update function of unity which needs to be worked around by the devs.

**Comment by asiaclonk** [comment]
Can confirm this happening with a connector attached to a freight station. A truck unloading cargo will randomly not fully unload to the connector and drive away with some cargo remaining.

**Reply by victorleblais** [reply]
The "unload" order unloads the vehicle for a specified amount of time or until the output is full.

Stations do not have any inventory so if you output using a connector, the connector will be full and thus the truck will leave.

If you want to make sure it unloads properly, simply tick "Unload all" to make sure the trucks stays until it's empty.

You can also build a warehouse next to your station to make a "buffer" for your truck to unload into, and then plug your conveyor into that warehouse.
